test/pythonml/integration_configuration.yaml

```diff
@@ -191,6 +192,14 @@ tests:
       - CLS_randomForest
       - POS_plotROC
 
+  Cls_ReadCSV_StandScale_GradientBoostedClassifier_Parity:
```

Comment:
Looks like there's a typo in the final unit here - we're running an ROC plot in this test, not a parity plot (see naming for the other classification test)

Reply:
Thanks James, I'll rename accordingly.